src/components/content/deployedServices/myServices/MyServiceHistory.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Table, Tag, Tooltip, Typography } from 'antd';
+import { ColumnsType } from 'antd/es/table';
+import React from 'react';
+import serviceModifyStyles from '../../../../styles/service-modify.module.css';
+import {
+    DeployRequest,
+    DeployedServiceDetails,
+    ServiceModificationAuditDetails,
+    VendorHostedDeployedServiceDetails,
+} from '../../../../xpanse-api/generated';
+import useListServiceModifyHistoryQuery from './query/useListServiceModifyHistoryQuery';
+
+const { Text } = Typography;
+
+export const MyServiceHistory = ({
+    deployedService,
+}: {
+    deployedService: DeployedServiceDetails | VendorHostedDeployedServiceDetails;
+}): React.JSX.Element => {
+    let serviceModificationAuditHistoryList: ServiceModificationAuditDetails[] = [];
+    const listServiceModifyHistoryQuery = useListServiceModifyHistoryQuery(deployedService.id);
+
+    if (listServiceModifyHistoryQuery.isSuccess && listServiceModifyHistoryQuery.data.length > 0) {
+        serviceModificationAuditHistoryList = listServiceModifyHistoryQuery.data;
+    }
+
+    const columns: ColumnsType<ServiceModificationAuditDetails> = [
+        {
+            title: 'ModifyId',
+            dataIndex: 'id',
+            align: 'center',
+            width: 100,
+        },
+        {
+            title: 'Previous',
```

Comment:
i think we can align the previous and new column as well to 'center' for uniformity